shared/src/mobileMain/kotlin/org/jetbrains/kotlinconf/ui/MarkdownUtils.kt

```diff
@@ -0,0 +1,172 @@
+package org.jetbrains.kotlinconf.ui
+
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.AnnotatedString
+import androidx.compose.ui.text.ParagraphStyle
+import androidx.compose.ui.text.SpanStyle
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.buildAnnotatedString
+import androidx.compose.ui.text.font.FontStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextDecoration
+import androidx.compose.ui.text.withStyle
+import androidx.compose.ui.unit.sp
+import org.intellij.markdown.MarkdownElementTypes
+import org.intellij.markdown.MarkdownElementTypes.ATX_2
+import org.intellij.markdown.MarkdownElementTypes.ATX_3
+import org.intellij.markdown.MarkdownElementTypes.ATX_4
+import org.intellij.markdown.MarkdownElementTypes.EMPH
+import org.intellij.markdown.MarkdownElementTypes.INLINE_LINK
+import org.intellij.markdown.MarkdownElementTypes.LINK_DESTINATION
+import org.intellij.markdown.MarkdownElementTypes.LINK_TEXT
+import org.intellij.markdown.MarkdownElementTypes.MARKDOWN_FILE
+import org.intellij.markdown.MarkdownElementTypes.PARAGRAPH
+import org.intellij.markdown.MarkdownTokenTypes
+import org.intellij.markdown.MarkdownTokenTypes.Companion.ATX_CONTENT
+import org.intellij.markdown.MarkdownTokenTypes.Companion.ATX_HEADER
+import org.intellij.markdown.MarkdownTokenTypes.Companion.BLOCK_QUOTE
+import org.intellij.markdown.MarkdownTokenTypes.Companion.COLON
+import org.intellij.markdown.MarkdownTokenTypes.Companion.EMPH
+import org.intellij.markdown.MarkdownTokenTypes.Companion.EOL
+import org.intellij.markdown.MarkdownTokenTypes.Companion.EXCLAMATION_MARK
+import org.intellij.markdown.MarkdownTokenTypes.Companion.HARD_LINE_BREAK
+import org.intellij.markdown.MarkdownTokenTypes.Companion.HTML_TAG
+import org.intellij.markdown.MarkdownTokenTypes.Companion.LBRACKET
+import org.intellij.markdown.MarkdownTokenTypes.Companion.LPAREN
+import org.intellij.markdown.MarkdownTokenTypes.Companion.RBRACKET
+import org.intellij.markdown.MarkdownTokenTypes.Companion.RPAREN
+import org.intellij.markdown.MarkdownTokenTypes.Companion.TEXT
+import org.intellij.markdown.MarkdownTokenTypes.Companion.WHITE_SPACE
+import org.intellij.markdown.ast.ASTNode
+import org.intellij.markdown.ast.getTextInNode
+import org.intellij.markdown.flavours.gfm.GFMFlavourDescriptor
+import org.intellij.markdown.parser.MarkdownParser
+import org.jetbrains.kotlinconf.ui.theme.JetBrainsSans
+
+class MarkdownStyle(

```

Comment:
have you tried https://github.com/mikepenz/multiplatform-markdown-renderer ?
I haven't but it looks ok